api/MediaKeySystemAccess.json

```diff
@@ -112,10 +112,12 @@
               "version_added": "13"
             },
             "firefox": {
-              "version_added": "43"
+              "version_added": "43",
+              "notes": "Before Firefox 50, returned configuration objects did not have a <code>distinctiveIdentifier</code> or <code>persistentState</code> property."
```

Comment:
Hmm. Two things here:

- The note feels a little ambiguous about what's missing. I'm not sure whether it means one of the two is missing (e.g., you'll get one or the other, but not both) or if neither show up. If the latter, I'd probably say something like "objects that have neither a … nor …" or have two notes, one for each property.
- Notes ought to be in present tense (e.g., "Before Firefox 50, returned objects do not have…")